docker/README.md

```diff
@@ -322,30 +322,30 @@ for configuring snapshot archival.
 Fiftyone Teams v2.6 introduces the ability to add a static banner to the
 application.
 
-Configure the Static Banner by setting the following environment variables in
-your `compose.override.yaml`.
-
-Banner text is configured with `FIFTYONE_APP_BANNER_TEXT`.
-
-Banner background color is configured with `FIFTYONE_APP_BANNER_COLOR`.
+Banner text is configured with
+`teams-cas.env.FIFTYONE_APP_BANNER_TEXT` and
+`teams-app.env.FIFTYONE_APP_BANNER_TEXT`.
+Banner background color is configured with
+`teams-cas.env.FIFTYONE_APP_BANNER_COLOR` and
+`teams-app.env.FIFTYONE_APP_BANNER_COLOR`.
+Banner text color is configured with
+`teams-cas.env.FIFTYONE_APP_BANNER_TEXT_COLOR` and
+`teams-app.env.FIFTYONE_APP_BANNER_TEXT_COLOR`.
```

Comment:
this isn't correct for docker-compose...

.env should be .environment ?

Reply:
Great catch.  Fixed.